corehq/apps/users/views/mobile/users.py

```diff
@@ -710,8 +710,7 @@ def check_username(self, in_data):
         exists = user_exists(full_username)
         if exists.exists:
             if exists.is_deleted:
-                result = {'warning': _('Username {} belonged to a user that was deleted.'
-                                       ' Reusing it may have unexpected consequences.').format(username)}
+                result = {'error': _('Username {} belonged to a user that was deleted.').format(username)}
```

Comment:
Open to suggestions for improvements to the wording.

Reply:
I think we should explicitly state "and cannot be reused" or similar for now. Alaska DoH wanted to make sure that it was clear that the system explicitly disallowed

Reply:
0d1e62ad0e4